test/specs/manifestBuilder.e2e.ts

```diff
@@ -120,11 +121,11 @@ describe('Manifest Builder', async () => {
     const workbench = await browser.getWorkbench();
     await utilities.getTextEditor(workbench, 'manifest.xml');
     // Clear output before running the command
-    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+    await utilities.clearOutputView(Duration.seconds(2));
```

Comment:
It looks like `clearOutputView()` sometimes has a duration of 1 second and sometimes has a duration of 2 seconds.  Would it make sense to standardize it to 1 second throughout the code, especially since that's the default timeout?

Reply:
I'll change it to 1 second